osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -229,6 +231,16 @@ protected bool UpdateJudgement(bool userTriggered)
         protected virtual void CheckForJudgements(bool userTriggered, double timeOffset)
         {
         }
+
+        public void Shake()
+        {
+            this.MoveToOffset(new Vector2(8, 0), 20).Delay(20)
```

Comment:
since you are using `MoveToOffset`, this will likely fail if you click rapidly (the additional offset will become incorrect).

i suggest using `MoveToX(Position + shake_amount)`

and yes, make `shake_amount` a function-local `const`

also use can use `.Then()` instead of `.Delay()`

also this should most definitely only be implemented in the osu! ruleset, not game-wide. Consider `DrawableOsuHitObject`